examples/neoIcoFoam/neoIcoFoam.cpp

```diff
@@ -64,29 +79,28 @@ int main(int argc, char* argv[])
                 }
             );
 
-        Info << "creating nf nu field" << endl;
         auto nuBCs = fvcc::createCalculatedBCs<fvcc::SurfaceBoundary<NeoN::scalar>>(rt.nfMesh);
         fvcc::SurfaceField<NeoN::scalar> nu(rt.exec, "nu", rt.nfMesh, nuBCs);
         NeoN::fill(nu.internalVector(), viscosity.value());
         NeoN::fill(nu.boundaryData().value(), viscosity.value());
 
-        Info << "creating nf phi field" << endl;
+        NeoN::Logging::info("creating nf phi field");
```

Comment:
`nf` is abbreviation of namespace used in the code.
For logging, it would be better to make it clear.

```suggestion
        NeoN::Logging::info("creating NeoFOAM phi field");
```